extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -939,7 +940,7 @@ export class AmpSlideScroll extends AMP.BaseElement {
         }
         dev()
           .assertElement(this.slideWrappers_[i])
-          .classList.remove(SHOWN_CSS_CLASS);
+          .classList.remove(ClassNames.SLIDES_ITEM_SHOW);
```

Comment:
This `dev().assertElement` may as well be removed/replaced with `?.`, particularly given the predicate on line 932 already accesses it without any sort of check `this.slideWrappers_[i].classList`

Reply:
Good call! Done